### PR TITLE
Fixes #31 Updates Issue and PR templates

### LINKED ISSUE #31: Update Issue and PR template
**Expected behavior**

Presently, Contributor has to do a lot of changes in Issue and PR  template. One has to remove lots of text present already in templates and then has to edit according to issue/PR.

**Observed behavior**
There should be simple templates so that contributor has to just mention the bug/feature instead of editing and removing lots of text.

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #31 

**Changes proposed in this Pull Request**
- Updates Issue and PR templates.

### CODE REVIEW

**Comment by ParthS007** [comment]
@aka-jain  Please review. Thanks

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,14 +1,23 @@
-_Thanks for taking the time to report a bug in the EvalAI project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/EvalAI-ngx/issues)._
+**I'm submitting a ...**  (check one with "x")
```
Comment (deshraj) [inline .github/ISSUE_TEMPLATE.md]: Can you please update the markdown with this:

```
**I'm submitting a**  

(Check one with "x" from given options)
- [ ] Bug report
- [ ] Feature request

## Current behavior
<!-- Describe how the bug manifests. -->

## Expected behavior
<!-- Describe what the behavior would be without the bug. -->

## Steps to reproduce
<!-- If you are able to illustrate the bug or feature request with an example, please provide steps to reproduce -->

## Related screenshots
<!-- Attach a screenshot of the issue here. -->

## Other information
<!-- List any other information that is relevant to your issue.-->

## System information
<!-- Add information about the system your facing this bug on. If you think this is irrelevant or a feature request, please remove this section -->

```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
-IMPORTANT NOTES (please read, then delete):
+<!--
+(Thanks for sending a pull request! Please fill out the blanks below.)
+-->
 
-* The PR title should start with "Fix #bugnum: " (if applicable), followed by a clear one-line present-tense summary of the changes introduced in the PR. For example: "Fix #bugnum: Introduce the first version of the collection editor.".
```
Comment (deshraj) [inline .github/PULL_REQUEST_TEMPLATE.md]: We definitely want to keep this message while creating a pull request.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->
+Fixes #
 
-* Please make sure to mention "#bugnum" somewhere in the description of the PR. This enables Github to link the PR to the corresponding bug.
+#### Checklist
```
Comment (deshraj) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think we don't need this section. Please drop this.

**Review by deshraj — CHANGES_REQUESTED** [review]

**Comment by ParthS007** [comment]
@deshraj  Please review. Thanks

**Review by deshraj — APPROVED** [review]
LGTM. :)

**Pull request merged**